targets/linenoise/wrapper.c

```diff
@@ -0,0 +1,5 @@
+#include "linenoise.c"
```

Comment:
why do you need to include .c file? Why can't you simply declare a linenoiseEdit function in fuzzer?